dash_core_components/Location.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, id=Component.REQUIRED, pathname=Component.UNDEFINED, search=C
         _locals.update(kwargs)  # For wildcard attrs
         args = {k: _locals[k] for k in _explicit_args if k != 'children'}
 
-        for k in ['id']:
+        for k in [u'id']:
```

Comment:
Where did this `u` (and in `Store` below) come from? Is this intentional, or a result of running the build script in py2 instead of py3?

Reply:
Just to wrap this up, the crux of the difference is that Python 3 does away with `unicode` and `str` is essentially `unicode` by default; in Python 2, importing via `json.load` will return Unicode strings. There are workarounds, and the one I've chosen to use is Mark Amery's  [`byteify`](https://stackoverflow.com/questions/956867/how-to-get-string-objects-instead-of-unicode-from-json/13105359#13105359) function.